core/src/operations/mod.rs

```diff
@@ -364,9 +364,8 @@ pub enum Operation {
 
     // ----- cryptographic operations -------------------------------------------------------------
     /// Applies a permutation of Rescue Prime Optimized to the top 12 elements of the stack. The
-    /// rate part of the sponge is assumed to be on top of the stack, and the capacity is expected
-    /// to be deepest in the stack, starting at stack[8]. For an RPO permutation of [A, B, C] where
-    /// A is the capacity, the stack should look like [C, B, A, ...] from the top.
+    /// elements at the top of the will be absorbed as the rate part of the sponge function
```

Comment:
nit: `Elements at the top of the stack`. Maybe we could textually specify the elements count instead of putting a rust range?

Reply:
do you mean something like it is done in the assembler?

https://github.com/0xPolygonMiden/miden-vm/blob/159a04f3d4648f77d664642409e521438378bd6c/assembly/src/assembler/instruction/crypto_ops.rs#L15-L23

I suppose if we go that route, I can update all the operations to have a consistent format for documenting it.